.circleci/config.yml

```diff
@@ -40,12 +40,13 @@ workflows:
       - lint-scripts
       - lint-charts
   sync:
-    triggers:
-      - schedule:
-          cron: "23,53 * * * *"
-          filters:
-            branches:
-              only:
-                - master
+    filters:
+      branches:
+        only:
+          - master
     jobs:
-      - sync
+      sync:
```

Comment:
Does `jobs` value need to be an array?

Reply:
Yeah.  I just simplified it some more and fixed that.